ios/Podfile.lock

```diff
@@ -1,399 +0,0 @@
-PODS:
```

Comment:
Remove this file from commit history please.

Reply:
This file is already deleted in my branch. Please let me know if i am missing something?

Reply:
Seems like the commit is still there.